pep-0609.rst

```diff
@@ -8,7 +8,7 @@ Author: Dustin Ingram <di@python.org>,
 Sponsor: Paul Ganssle <paul@ganssle.io>
 BDFL-Delegate: TBD
 Discussions-To: https://discuss.python.org/t/pep-609-pypa-governance/
-Status: Draft
+Status: Accepted
```

Comment:
```suggestion
Status: Active
```

Reply:
Does this mean we're already ready to implement the listed "changes would better accommodate the PyPA’s usage of the PEP process" in https://github.com/python/steering-council/issues/30?